compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -342,7 +342,7 @@ struct EncodeTensorConstantOp
              << alignedType << " does not have integral number of total bytes";
     }
     rewriter.replaceOpWithNewOp<IREE::Stream::AsyncConstantOp>(
-        op, op.getResult().getType(), encodedAttr, resultSize,
+        op, op.getResult().getType(), Value(), encodedAttr, resultSize,
```

Comment:
It is better to add argument comment.

```suggestion
        op, op.getResult().getType(), /*await_timepoint=*/Value(), encodedAttr, resultSize,
```